src/components/tab-title/tab-title.scss

```diff
@@ -93,10 +88,6 @@ span {
   border-block-end-style: solid;
 }
 
-span {
```

Comment:
I'm not sure why `span` was here. No span is being rendered in the component. cc @macandcheese @paulcpederson